lib/dotcom/cache/trip_plan_feedback/cache.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Dotcom.Cache.TripPlanFeedback.Cache do
+  use Nebulex.Cache,
+    otp_app: :dotcom,
+    adapter: NebulexRedisAdapter
```

Comment:
I really wanted to use this and it turns out it doesn't behave the same way - it'll let any arbitrary object get cached instead of checking the argument value.